src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -783,23 +783,22 @@ namespace rpc {
   };
 
   //-----------------------------------------------
-  OXEN_RPC_DOC_INTROSPECT
-  struct GET_NET_STATS : LEGACY
+  /// Retrieve general information about the state of the network.
+  ///
+  /// Inputs: none.
+  ///
+  /// Output values available from a restricted/admin RPC endpoint:
+  ///
+  /// \p status General RPC status string. `"OK"` means everything looks good.
```

Comment:
Should add a `-` on the beginning of each of these so that doxygen will pick them up as list items.  I think.